schemas/djornl/djornl_node.yaml

```diff
@@ -1,87 +1,107 @@
-name: djornl_gene
+name: djornl_node
```

Comment:
There are a couple of things going on here. The front end is going to display all node types in a single table, so from that perspective, it is better to have a uniform structure for nodes. As you said, we don't currently have different fields in the phenotypes and genes, and I really want to get the Jacobson group to revise their node data so that it's in a more normalised format, instead of a hodgepodge of concatenated columns and unstructured text. It was also much easier to make all the queries return results in a uniform format if we use a single table.

==> I am looking to revisit the collections after discussion with the Jacobson people.